### PR TITLE
Feature/175385849 add nonroot docker image

### PR DESCRIPTION
**Pull request by RyanRHall**
Addresses [#175385849](https://www.pivotaltracker.com/story/show/175385849)

### CODE REVIEW

**Review by topliceanu — COMMENTED** [review]
Looks good! Next, I think the "release" tooling needs to be upgraded to build both images.

**Review by topliceanu — APPROVED** [review]
lgtm

File: core/chainlink-nonroot.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y ca-certificates
+
+COPY --from=1 /go/bin/chainlink /usr/local/bin/
+
+EXPOSE 6688
+
+RUN useradd --uid 14933 --create-home chainlink
+USER chainlink
```
Comment (HenryNguyen5) [inline core/chainlink-nonroot.Dockerfile:76]: Are these the only two differences between root and non-root? If so, we should try to inject these as build-args instead, and continue maintaining one file.

**Review by HenryNguyen5 — CHANGES_REQUESTED** [review]

File: core/chainlink-nonroot.Dockerfile (hunk 1 above)
Comment (RyanRHall) [reply core/chainlink-nonroot.Dockerfile:76]: @HenryNguyen5 I think the plan is to deprecate the old root image after node operators switch over. Might be easier to just keep two files and then delete the old one. Thoughts? I don't feel strongly either way.

Comment (HenryNguyen5) [reply core/chainlink-nonroot.Dockerfile:76]: Yeah, its like a 3 month period though, i'd imagine we'd change the dockerfile more often than that. We used to have another file for SGX that took awhile to get removed and was a pain to maintain

Comment (RyanRHall) [reply core/chainlink-nonroot.Dockerfile:76]: sg man! I'll add the build args.

**Review by HenryNguyen5 — COMMENTED** [review]
Looks great, nice job on this!

**Review by HenryNguyen5 — APPROVED** [review]

**Pull request merged**